src/frontend/src/components/AccessibilityHelper.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { css } from '@/styled-system/css'
+
+const skipLinkStyles = css({
+  position: 'absolute',
+  top: '-40px',
+  left: '6px',
+  background: '#000',
+  color: '#fff',
+  padding: '8px',
+  textDecoration: 'none',
+  borderRadius: '4px',
+  zIndex: 1000,
+  '&:focus': {
+    top: '6px',
+  },
+})
+
+const announcementStyles = css({
+  position: 'absolute',
+  left: '-10000px',
+  width: '1px',
+  height: '1px',
+  overflow: 'hidden',
+})
+
+export const AccessibilityHelper = () => {
```

Comment:
Hey, thanks for this :)

What do you think about splitting this in two different components? I admit it feels a bit weird to have skip links and route announcer in the same component, they are not really related from my pov.